buildconfig/pygame-stubs/math.pyi

```diff
@@ -124,6 +124,15 @@ class Vector2:
     def __rmul__(self, other: float) -> Vector2: ...
     def __truediv__(self, other: float) -> Vector2: ...
     def __floordiv__(self, other: float) -> Vector2: ...
+    def __neg__(self) -> Vector2: ...
+    def __pos__(self) -> Vector2: ...
+    def __bool__(self) -> bool: ...
```

Comment:
is it necessary to type this? Python's `object` has `__bool__` already, which is presumably typed

Reply:
I don't think it will do any harm in the type stub,